### PR TITLE
micro refactor

### PR DESCRIPTION
**Pull request by vict0rsch**
* add git hash logging
* clean imports
* move comet args to command-line args

### CODE REVIEW

File: scripts/train.py [hunk 1]
```diff
@@ -45,13 +46,15 @@
...
-    help="output of git log --pretty=format:'%h' -n 1",
+    "--project", type=str, default="testing-munit", help="Comet's project_name"
 )
+parser.add_argument(
+    "--workspace", type=str, default="sunandr", help="Comet's workspace"
+)
+
 opts = parser.parse_args()
 
+comet_exp = Experiment(workspace=opts.workspace, project_name=opts.project)
```
Comment (vict0rsch) [inline scripts/train.py]: Notable change

File: scripts/train.py [hunk 2]
```diff
@@ -62,7 +65,7 @@
 
 if comet_exp is not None:
     comet_exp.log_asset(file_data=opts.config, file_name=Path(opts.config))
-    comet_exp.log_parameter("git_hash", opts.git_hash)
+    comet_exp.log_parameter("git_hash", get_git_revision_hash())
```
Comment (vict0rsch) [inline scripts/train.py]: Log git hash

**Comment by 51N84D** [comment]
Just discovered the "Reproduce" feature in comet. Seems related to this

**Comment by vict0rsch** [comment]
@51N84D

**Comment by vict0rsch** [comment]
test @51N84D

**Comment by vict0rsch** [comment]
@51N84D I just added extra-adam

**Comment by vict0rsch** [comment]
When you merge, control the optimizer with 

```
optimizer: extraadam # or adam
```

**Pull request merged**